keyboards/numatreus/config.h

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2019 yohei <yohewi@gmail.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0xE80A
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    yohewi
+#define PRODUCT         NumAtreus
+#define DESCRIPTION     QMK keyboard firmware for NumAtreus
```

Comment:
`DESCRIPTION` is usually used to describe the keyboard. It's the product description send to the computer, describing what USB device was just inserted.

Something like "An 80% ergonomic keyboard" may make more sense.

That's all personal taste though :)

Reply:
Not quite. You won't see this text unless you use the Command feature. Otherwise, it's completely useless.